### ISSUE TITLE
"Class Behat\Gherkin\Cache\FileCache does not exist" after committing to Git on Windows

### ISSUE THREAD

**Issue by jamesgrobertson** [jhedstrom/drupalextension#152]
I'm able to install the Drupal Extension fine, but when I commit it to a Git branch on Windows, switch branches, and then try to check out that branch again later it doesn't work anymore, giving me:

    [ReflectionException]
    Class Behat\Gherkin\Cache\FileCache does not exist

Also happens when cloning the repo and trying to check out the branch with the extension code in it.

I realize this might be a Git or file permissions issue, I'm just hoping someone else has had this problem and can point me to how they resolved it.

**Comment by jhedstrom** [comment]
Does running with `--verbose` provide any additional insight? I also wonder if a `git status` shows untracked files that perhaps disappear before or after branch switching?

**Comment by jamesgrobertson** [comment]
tl;dr: I fixed it, most likely by not running my Git shell as an administrator.

To try and fix it, I tried installing in an Ubuntu environment and committing from there. That install didn't work on Windows, so I tried re-installing on Windows, but without admin permissions for Git. I'm now able to use it on Windows and switch branches, etc. Thanks for replying!
